python/pandemic_simulator/environment/regulation_stages.py

```diff
@@ -1,63 +1,76 @@
-                       },
-                       stage=3),
-    PandemicRegulation(stay_home_if_sick=True,
-                       practice_good_hygiene=True,
-                       wear_facial_coverings=True,
-                       social_distancing=0.7,
-                       risk_to_avoid_gathering_size={Risk.HIGH: 0, Risk.LOW: 0},
-                       location_type_to_rule_kwargs={
-                           Office: {'lock': True},
-                           School: {'lock': True},
-                           BarberShop: {'lock': True},
-                           RetailStore: {'lock': True},
-                       },
-                       stage=4)
+austin_regulations: List[CovidRegulation] = [
+    CovidRegulation(stage=0),
+    CovidRegulation(stay_home_if_sick=True,
+                    practice_good_hygiene=True,
+                    wear_facial_coverings=False,
+                    social_distancing=DEFAULT,
+                    risk_to_avoid_gathering_size={Risk.HIGH: 25, Risk.LOW: 50},
+                    location_type_to_rule_kwargs={
+                        Office: {'lock': False},
+                        School: {'lock': False},
+                        BarberShop: {'lock': False},
+                        RetailStore: {'lock': False},
+                        Restaurant: {
+                            'open_time': SimTimeTuple(hours=(12, 1, 20, 21, 22), week_days=tuple(range(1, 7)))},
+                        Bar: {'lock': False},
+                    },
+                    stage=1),
+    CovidRegulation(stay_home_if_sick=True,
+                    practice_good_hygiene=True,
+                    wear_facial_coverings=True,
+                    social_distancing=0.3,
+                    risk_to_avoid_gathering_size={Risk.HIGH: 10, Risk.LOW: 25},
+                    location_type_to_rule_kwargs={
+                        Office: {'lock': False},
+                        School: {'lock': True},
+                        BarberShop: {'lock': True},
+                        RetailStore: {'lock': False},
+                        Restaurant: {
+                            'open_time': SimTimeTuple(hours=(12, 1, 20, 21, 22), week_days=tuple(range(1, 7)))},
+                        Bar: {'lock': False},
+                    },
+                    stage=2),
+    CovidRegulation(stay_home_if_sick=True,
+                    practice_good_hygiene=True,
+                    wear_facial_coverings=True,
+                    social_distancing=0.5,
+                    risk_to_avoid_gathering_size={Risk.HIGH: 0, Risk.LOW: 0},
+                    location_type_to_rule_kwargs={
+                        Office: {'lock': False},
+                        School: {'lock': True},
+                        BarberShop: {'lock': True},
+                        RetailStore: {'lock': False},
+                        Restaurant: {
+                            'open_time': SimTimeTuple(hours=(12, 1, 20, 21, 22), week_days=tuple(range(1, 7)))},
```

Comment:
are these `open_times` observed in Austin? Because I think the Austin's regulations don't specify any reduced hours in the stage plan AFAIK.

Reply:
I believe Vijay had the same hours for all of them. He just explicitly set it every time. Setting it like this isn't necessary though right?

Reply:
Yeah, it is not necessary.